shell.js

```diff
@@ -42,69 +44,92 @@ process.argv.forEach(function(val, index, array) {
 });
 
 if (opts.brain === undefined) {
-	console.log("Usage: node shell.js [--debug --utf8] </path/to/brain>");
+	console.log("Usage: node shell.js [--debug --utf8 --watch] </path/to/brain>");
 	process.exit(1);
 }
 
 //------------------------------------------------------------------------------
 // Initialize the RiveScript bot and print the welcome banner.
 //------------------------------------------------------------------------------
 
-var bot = new RiveScript({
-	debug: opts.debug,
-	utf8:  opts.utf8,
-});
-bot.loadDirectory(opts.brain, loading_done, loading_error);
-
-function loading_done(batch_num) {
-	bot.sortReplies();
+var bot = null;
+function loadBot() {
+	bot = new RiveScript({
+		debug: opts.debug,
+		utf8:  opts.utf8,
+	});
+	bot.ready = false;
+	bot.loadDirectory(opts.brain, loading_done, loading_error);
+}
+loadBot();
 
-	console.log("      .   .       ");
-	console.log("     .:...::      RiveScript Interpreter (JavaScript)");
-	console.log("    .::   ::.     Library Version: v" + bot.version());
-	console.log(" ..:;;. ' .;;:..  ");
-	console.log("    .  '''  .     Type '/quit' to quit.");
-	console.log("     :;,:,;:      Type '/help' for more options.");
-	console.log("     :     :      ");
-	console.log("");
-	console.log("Using the RiveScript bot found in: " + opts.brain);
-	console.log("Type a message to the bot and press Return to send it.");
-	console.log("");
+if (opts.watch) {
+	fs.watch(opts.brain, {recursive: false}, function() {
+		console.log("");
+		console.log('[INFO] Brain changed, reloading bot.');
+		rl.prompt();
```

Comment:
Nope it's just to display the `>` input sign again in the terminal's last line. Necessary as we've been writing to the `console.log`